warnings.rb

```diff
@@ -0,0 +1,10 @@
+
+a = { "hello" => "world" }
+
+2 + 5
+
+
+
+puts "Yo"
```

Comment:
_[farcy v0.1b](https://github.com/appfolio/farcy)_

* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.